src/krati_Jain/Assignment_14/CalculatorUserInput.java

```diff
@@ -0,0 +1,47 @@
+package krati_Jain.Assignment_14;
+
+import java.util.Scanner;
+
+public class CalculatorUserInput {
+	
+		void numberInput() {
+			Scanner scan = new Scanner(System.in);
```

Comment:
We should not ask, user to give inputs after calling methods, 
We should ask user to give input while calling methods, and we take input once only from main method not in every method